RecentActivity/src/org/sleuthkit/autopsy/recentactivity/RAImageIngestModule.java

```diff
@@ -227,4 +227,15 @@ protected static String getRAOutputPath(Case a_case, String mod) {
         }
         return tmpDir;
     }
+    
+    /**
+     * Get relative path for module output folder.
+     *
+     * @throws NoCurrentCaseException if there is no open case.
+     * @return the relative path of the module output folder
+     */
+    static String getRelModuleOutputPath() throws NoCurrentCaseException {
+        return Case.getCurrentCaseThrows().getModuleOutputDirectoryRelativePath() + File.separator
+                + "RecentActivity";
```

Comment:
It's a little cleaner to build the path using Paths.  Then you don't have to worry about File.separator.

You need the \\NON-NLS at the end of this line or you could use "OpenIDE-Module-Name" from bundles.properties.